guides/source/upgrading_ruby_on_rails.md

```diff
@@ -76,6 +76,17 @@ Rails 5.2 adds bootsnap gem in the [newly generated app's Gemfile](https://githu
 The `app:update` task sets it up in `boot.rb`. If you want to use it, then add it in the Gemfile,
 otherwise change the `boot.rb` to not use bootsnap.
 
+### Expirity is now embedded in the cookies values
```

Comment:
`Expiry` … `in signed or encrypted cookies`. It's only those cookies that get it embedded.